integration-tests/utils/integration-helpers.js

```diff
@@ -27,38 +27,56 @@ const clipToken = str => str.toString().substr(-10, 10)
 const testWorkspaceNamePrefix = 'terra-ui-test-workspace-'
 const getTestWorkspaceName = () => `${testWorkspaceNamePrefix}${uuid.v4()}`
 
+
 const makeWorkspace = withSignedInPage(async ({ page, billingProject }) => {
   const workspaceName = getTestWorkspaceName()
   try {
-    await page.evaluate((name, billingProject) => {
-      return window.Ajax().Workspaces.create({ namespace: billingProject, name, attributes: {} })
+    const response = await page.evaluate((name, billingProject) => {
+      return new Promise(async (resolve, reject) => {
+        return await window.Ajax().Workspaces.create({ namespace: billingProject, name, attributes: {} })
+          .then(resp => resolve(resp))
+          .catch(async err => reject(await err.text()))
+      })
```

Comment:
This looks like it could be simplified.

```suggestion
    const response = await page.evaluate(async (name, billingProject) => {
      try {
        return await window.Ajax().Workspaces.create({ namespace: billingProject, name, attributes: {} })
      } catch (err) {
        const message= await err.text()
        throw message
      }
    })
```

Reply:
I drafted a nearly identical comment, but I wasn't 100% sure. I agree that simplification here would be nice.